packages/babel-plugin-transform-es2015-classes/src/index.js

```diff
@@ -51,7 +53,7 @@ export default function({ types: t }) {
         node[VISITED] = true;
 
         let Constructor = VanillaTransformer;
```

Comment:
This `Constructor` assignment is can be moved to the top since it just uses `loose`.